frontend/pages/label/[id].vue

```diff
@@ -146,14 +154,19 @@
                 <MdiDelete class="mr-2" />
                 Delete
               </BaseButton>
+              <BaseButton class="btn btn-sm">
```

Comment:
toggle button on the label detail page to hide the total purchase price value in the header